docs/manual/existing_performance_counters.qbk

```diff
@@ -126,6 +126,18 @@ system and application performance.
         [Returns the overall execution time of all AGAS services provided by the
          given AGAS service category since its creation (in nanoseconds).]
     ]
+    [   [`/agas/count/entries`]
+        [`locality#*/total`
+
+          where:[br] `*` is the locality id of the locality the AGAS cache should
+          be queried. The locality id is a (zero based) number identifying the
+          locality.
+        ]
+        [None]
+        [Returns the number of cache entries resident in the AGAS cache of the
+        specified locality (see
+         `<cache_statistics>`.]
+    ]
```

Comment:
On 02/12/2016 01:41 PM, Hartmut Kaiser wrote:
> In docs/manual/existing_performance_counters.qbk
> <https://github.com/STEllAR-GROUP/hpx/pull/1984#discussion_r52736540>:
> 
>> @@ -126,6 +126,18 @@ system and application performance.
>>          [Returns the overall execution time of all AGAS services provided by the
>>           given AGAS service category since its creation (in nanoseconds).]
>>      ]
>> +    [   [`/agas/count/entries`]
>> +        [`locality#*/total`
>> +
>> +          where:[br] `*` is the locality id of the locality the AGAS cache should
>> +          be queried. The locality id is a (zero based) number identifying the
>> +          locality.
>> +        ]
>> +        [None]
>> +        [Returns the number of cache entries resident in the AGAS cache of the
>> +        specified locality (see
>> +         `<cache_statistics>`.]
>> +    ]
> 
> Thanks! Do we still have the NUMA domain specific counters? If yes, then
> it should be reflected in the docs. If not, the we can remove the
> NUMA-counter discoverer functions.

There are no NUMA specific counters available at the moment. I'd suggest
to leave the expansion/discovery in for now, might come in handy
eventually, for example for inter/intra NUMA memory traffic counters or so.

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/STEllAR-GROUP/hpx/pull/1984/files#r52736540>.
> 


-- 
Thomas Heller
Friedrich-Alexander-Universität Erlangen-Nürnberg
Department Informatik - Lehrstuhl Rechnerarchitektur
Martensstr. 3
91058 Erlangen
Tel.: 09131/85-27018
Fax:  09131/85-27912
Email: thomas.heller@cs.fau.de